server/pulp/server/db/model/__init__.py

```diff
@@ -359,7 +359,10 @@ def _write_pem_file(self, config_key, path):
                 misc.mkdir(os.path.dirname(self._pki_path))
                 os.mkdir(self._pki_path, 0700)
             with os.fdopen(os.open(path, os.O_WRONLY | os.O_CREAT, 0600), 'w') as pem_file:
-                pem_file.write(self.config[config_key].encode('utf-8'))
+                if self.config[config_key] is unicode:
```

Comment:
I don't think this code would work.  ```isinstance(config_key, unicode)``` would be better, I think.

@goosemania do you agree?